### PR TITLE
Draft content for our Enterprise Reporting release

### PR DESCRIPTION
**Pull request by danjohansenconsulting**
New functionality that is being provided for embedded dashboard reporting and data download.  Available to our Enterprise customers.   A separate PR will be issued for the Enterprise API page where data can be programmatically retrieved.

We are anticipating a launch date of tomorrow, April 16, so that is the timeline we'll be looking to finalize and publish this change.

### CODE REVIEW

File: docs/guides/cloud/analytics/enterprise-reporting.mdx [hunk 1]
```diff
@@ -4,46 +4,103 @@ sidebar_position: 30
...
 
-A new tool designed for technical experts to facilitate informed
-decision-making and easily highlight the value of testing
-initiatives to leadership.
+If available in your Cypress plan, Enterprise Reporting is accessible by clicking on the "Enterprise
+Reporting" link in the menu.
 
-:::tip
+Your insights are logically grouped into a number of tabs such as "Overall Usage", "Test Results", "Test
+Suite", etc. When the page first loads, data is displayed for all of the current month and the preceeding
```
Comment (MikeMcC399) [inline docs/guides/cloud/analytics/enterprise-reporting.mdx:28]:
```suggestion
Suite", etc. When the page first loads, data is displayed for all of the current month and the preceding
```

https://www.merriam-webster.com/dictionary/preceding

File: docs/guides/cloud/analytics/enterprise-reporting.mdx [hunk 2]
```diff
@@ -4,46 +4,103 @@ sidebar_position: 30
...
+test suite in various projects? These are some of the questions that will be answered by the data presented on this tab.
+
+<DocsImage
+  src="/img/guides/cloud/analytics/enterprise-reporting-test-suite.png"
+  alt="Cloud Enterprise Reporting test suite report"
+/>
+
+### Download data from Cloud
+
+All of the charts shown in Enteprise Reporting allow you to download the raw data used to
```
Comment (MikeMcC399) [inline docs/guides/cloud/analytics/enterprise-reporting.mdx:78]:
```suggestion
All of the charts shown in Enterprise Reporting allow you to download the raw data used to
```

**Review by MikeMcC399 — CHANGES_REQUESTED** [review]
Typos

**Review by MikeMcC399 — APPROVED** [review]
typos are fixed. thanks!

File: docs/guides/cloud/analytics/enterprise-reporting.mdx [hunk 3]
```diff
@@ -4,46 +4,103 @@ sidebar_position: 30
 sidebar_custom_props: { 'new_label': true }
 ---
 
-:::info
+Enterprise Reporting is available within Cypress Cloud so that you can spot trends and anomalies,
+track usage, and govern your overall test program across all projects in your organization. Included
+is the ability to interact with visual cards that highlight usage, test metrics, test types, flake,
+build versions, and more. The raw data from all graphs presented in Cloud can be downloaded in a variety
+of formats. And if you prefer to programatically retrieve test data for use within your own analytics
```
Comment (MikeMcC399) [inline docs/guides/cloud/analytics/enterprise-reporting.mdx:11]:
```suggestion
of formats. And if you prefer to programmatically retrieve test data for use within your own analytics
```

Elsewhere in this repo and on https://www.merriam-webster.com/dictionary/programmatically it's spelled with two m characters.

**Review by elevatebart — APPROVED** [review]

**Pull request merged**